include/swift/Frontend/Frontend.h

```diff
@@ -433,6 +434,64 @@ class CompilerInstance {
   /// Frees up the ASTContext and SILModule objects that this instance is
   /// holding on.
   void freeContextAndSIL();
+
+private:
+  bool loadStdlibAndMaybeSwiftOnoneSupport();
+  ModuleDecl *importUnderlyingModule(ClangImporter *clangImporter);
+  ModuleDecl *importBridgingHeader(ClangImporter *clangImporter);
```

Comment:
It doesn't really make sense to pass the importer as an argument here. It's something you can get from the ASTContext, which is already owned by the CompilerInstance.

Reply:
Done!